### PR TITLE
[TASK] Triskelion spinning endlessly on chart (only with not activated w...

### PR DESCRIPTION
**Pull request by Mesrop**
...allets) (RT-3164)

### CODE REVIEW

File: src/jade/tabs/widgets/markets.jade [hunk 1]
```diff
@@ -51,5 +51,5 @@
       counter-currency-issuer="{{ issuerNameToAddress(secondIssuerSelected, secondCurrencySelected) }}"
       )
 
-    div.widget_loader(ng-show="showWidgetAssetAllocation && !exchangeRatesNonempty")
```
Comment (vhpoet) [inline src/jade/tabs/widgets/markets.jade]: why did you remove `showWidgetAssetAllocation`?

Comment (vhpoet) [inline src/jade/tabs/widgets/markets.jade]: @Mesrop

**Comment by Mesrop** [comment]
@vhpoet can you check pls

**Comment by jublo** [comment]
@Mesrop This needs a rebase.

**Comment by vhpoet** [comment]
LGTM

**Comment by mrajvanshy** [comment]
LGTM
